fed/api.py

```diff
@@ -55,6 +55,7 @@ def init(
     cross_silo_serializing_allowed_list: Dict = None,
     exit_on_failure_cross_silo_sending: bool = False,
     cross_silo_messages_max_size_in_bytes: int = None,
+    enable_cross_silo_ping: bool = False,
```

Comment:
```suggestion
    enable_waiting_for_other_parties_ready: bool = False,
```
FYI: Just asked chatgpt for a good variable name~~~

Reply:
Nice suggestion!